src/components/App/index.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import './App.css';
 import Footer from '../Footer/';
-import Content from '../ResourceCard/Example';
 import ClusterResources from '../ClusterResources/';
+import NavBar from '../NavBar/';
 
 /* These below are for showing purposes */
 /* THIS IS GOING TO BE REMOVED IN THE DASHBOARD FEATURE */
 
 function App() {
   return <div>
+    <NavBar />
     <div className="Disposable">
       <div><ClusterResources /></div>
-    </div>
+    </div>;
     <Footer />
   </div>
```

Comment:
Colon should be here
```suggestion
  </div>;
```

Reply:
@Mubangizi this has been fixed.